actor/src/main/scala/org/apache/pekko/serialization/Serialization.scala

```diff
@@ -484,7 +484,7 @@ class Serialization(val system: ExtendedActorSystem) extends Extension {
   // include "com.google.protobuf.GeneratedMessage" = proto in configured serialization-bindings.
   private def checkGoogleProtobuf(className: String): Boolean = checkClass("com.google.protobuf", className)
 
-  // akka-protobuf is now not a dependency of remote so only load if user has explicitly added it
+  // pekko-tobuf is now not a dependency of remote so only load if user has explicitly added it
```

Comment:
```suggestion
  // pekko-protobuf is now not a dependency of remote so only load if user has explicitly added it
```

Reply:
Seems still broken?

Reply:
Not sure, GH seems to show an old version to me...

Reply:
I think this is fixed now - I've just rebased the commits and removed the merge commits - and force pushed